core/server/master/src/main/java/alluxio/master/block/meta/WorkerTimeMeta.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.master.block.meta;
+
+/**
+ *An object representation of the worker metadata.This class is not thread safe
+ *so accessing or updating the fields need to use AtomicReference<WorkerTimeMeta>.
+ */
+public class WorkerTimeMeta {
+  /** the start time of worker. */
+  private long mStartTime;
```

Comment:
This is not even used. 
Also, I don't see how this will be different than registration time.